### PR TITLE
update middleware monitoring vars

### PR DESCRIPTION
**Pull request by JameelB**
## Additional Information
Update the middleware monitoring vars to maintain naming consistency in the manifest file.

## Verification Steps
1. Run the installer
2. Ensure installer runs successfully and the monitoring middleware is  installed.

### CODE REVIEW

File: evals/inventories/group_vars/all/manifest.yaml [hunk 1]
```diff
@@ -79,13 +79,15 @@ webapp_operator_resources: 'https://raw.githubusercontent.com/integr8ly/tutorial
...
 #controls whether nexus is installed or not
 nexus: true
 nexus_version: '2.14.11-01'
 
-monitoring_label_name: "monitoring-key"
-monitoring_label_value: "middleware"
-middleware_monitoring_operator_version: '0.0.3'
+monitoring_label_name: 'monitoring-key'
+monitoring_label_value: 'middleware'
+middleware_monitoring_operator_release_tag: '0.0.3'
```
Comment (pb82) [inline evals/inventories/group_vars/all/manifest.yaml]: Looks like `0.0.4` is the latest release: https://github.com/integr8ly/application-monitoring-operator/releases

There's an automation PR to bump that version: https://github.com/integr8ly/installation/pull/412/files

Comment (mikenairn) [reply evals/inventories/group_vars/all/manifest.yaml]: @pb82 That was just a test release we did yesterday. @JameelB We should probably delete that release, the branch and the PR.

**Comment by mikenairn** [comment]
@JameelB These changes look good to me, but i guess we should verify it.

File: evals/inventories/group_vars/all/manifest.yaml (hunk 1 above)
Comment (JameelB) [reply evals/inventories/group_vars/all/manifest.yaml]: @pb82 @mikenairn i've closed the PR yesterday but will delete the branch and release now

**Pull request merged**